addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -541,8 +545,7 @@ function makeActionManager(env) {
         }
 
         viewProps.noBreadcrumbs =
-            "no_breadcrumbs" in action.context ? action.context.no_breadcrumbs : target === "new";
-        delete action.context.no_breadcrumbs;
+            "_noBreadcrumbs" in action ? action._noBreadcrumbs : target === "new";
```

Comment:
I'm curious: why wasn't no_breadcrumbs still in the context here?

Reply:
I suppose the original API is wrong, and that it should have been a field of the action itself.
If we don't delete no_breadcrumbs from the context, it will be sent in all RPC to the server, this could (but doesn't in my opinion) have some side effects:
- sending irrelevant keys in context to the server
- alter the behavior of python function calls if some code use the key (but again, no one should, that key is irrelevant to python)
- cache miss python/js for cached (orm cache or otherwise) functions, when the context is part of the cache key

Reply:
Ok nevermind, I didn't see ligne 545 :D So you can indeed ignore my 2 comments :D